### PR TITLE
Update config for 1.8 release.

### PR DESCRIPTION
**Pull request by steveperry-53**
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/kubernetes.github.io/5661)

### CODE REVIEW

**Comment by steveperry-53** [comment]
@chenopis, I'm stabbing in the dark here. Have I done the right thing?

File: _config.yml [hunk 1]
```diff
@@ -23,6 +23,11 @@ defaults:
       githubbranch: "master"
       docsbranch: "master"
       versions:
+        - fullversion: "v1.8.0"
+          version: "v1.8"
+          githubbranch: "release-1.8"
```
Comment (chenopis) [inline _config.yml]: set *githubbranch* to `v1.8.0`

File: _config.yml [hunk 2]
```diff
@@ -23,6 +23,11 @@ defaults:
...
+          docsbranch: "release-1.8"
+          url: https://kubernetes.io/docs/home/
         - fullversion: "v1.7.3"
```
Comment (chenopis) [inline _config.yml]: For now, set fullversion to `v1.7.6` as per the most current version in the [CHANGELOG](https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG.md).

Also, set the *url* for v1.7.x to `https://kubernetes-io-v1-7.netlify.com/docs/home/` until we can get @thockin to setup a subdomain.

File: _config.yml (hunk 1 above)
Comment (steveperry-53) [reply _config.yml]: Done, but I don't understand what branch that is.

File: _config.yml (hunk 2 above)
Comment (steveperry-53) [reply _config.yml]: Done

**Comment by steveperry-53** [comment]
@chenopis Added a commit that addresses your comments.

File: _config.yml (hunk 1 above)
Comment (chenopis) [reply _config.yml]: It's the GitHub branch from the main Kubernetes repo. If you do a search in the docs, you can see, for example, that in `/docs/admin/authorization/abac.md`, there is a link that uses that variable: 

`[Complete file example](http://releases.k8s.io/{{page.githubbranch}}/pkg/auth/authorizer/abac/example_policy_file.jsonl)`

**Comment by chenopis** [comment]
*githubbranch* is the version tag used in the main Kubernetes repo. If you do a search in the docs, you can see, for example, that in `/docs/admin/authorization/abac.md`, there is a link that uses that variable: 

`[Complete file example](http://releases.k8s.io/{{page.githubbranch}}/pkg/auth/authorizer/abac/example_policy_file.jsonl)`

If then you go to https://github.com/kubernetes/kubernetes/blob/v1.7.6/pkg/auth/authorizer/abac/example_policy_file.jsonl, you can look at the list of versions in the tag drop down.

screen shot 2017-09-27 at 11 13 57 pm

**Pull request merged**